### PR TITLE
Make 'run' parameter configurable

### LINKED ISSUE #69: Add ability set run param via cfg
I'd like the default behavior to be to set `run = False` on found issue, instead of explicitly setting this per issue.
It would be great if this value would be exposed in cfg as well.

### PR DESCRIPTION
**Pull request by lukas-bednar**
Fixes #69

### CODE REVIEW

**Comment by lukas-bednar** [comment]
@liiight feel free to comment

**Comment by liiight** [comment]
Looks great to me!

File: README.rst [hunk 1]
```diff
@@ -30,6 +30,10 @@ Test results
    -  the test is executed and based on it
       the result is **passed** or **failed**
 
+**NOTE:** You can set default value for ``run`` parameter globally in config
+file (*jira_run_test_case*) or from CLI *--jira-do-not-run-test-case*.
```
Comment (lukas-bednar) [inline README.rst]: I will change *something* to ``something`` it is easier to read.

**Comment by liiight** [comment]
Awesome, thanks for this!
Any idea when version will be released?

**Comment by lukas-bednar** [comment]
@liiight I want to close that second issue related to skipped info #68 .
I have issue to display it properly in both, terminal & xunit report.
Anyway I will release it in the worst case by end of week.

**Comment by liiight** [comment]
That's great to hear it'll be released soon, I'll be able to remove all those `run = False` I added 😉 
A shame about #68 though, that would have made my reports flawless. I'm using [pytest-html](https://github.com/pytest-dev/pytest-html) and I customized it a lot, catching the xfail reason would have been super nice.
Anything I can help with? I'd gladly contribute in any way that's helpful.

**Comment by lukas-bednar** [comment]
@liiight  here is what I have so far #71 .
But it doesn't behave consistently for all cases, xpass/xfail/skipped.
Feel free to take a look and play with it.

**Pull request merged**